_pkgdown.yml

```diff
@@ -2,3 +2,95 @@ navbar:
   right:
     - icon: fa-github
       href: https://github.com/NCEAS/arcticdatautils
+reference:
+
+-title: Functions
+
+- subtitle: Access functions
+  desc: Functions to change who can access the package
+  contents:
+  - starts_with("add_")
+
+- subtitle: DataOne functions
+  desc: Helper functions for the dataone package
+  contents:
+  - 'is_token_set'
+  - 'get_token'
+  - 'is_token_expired'
+  - 'get_mn_base_url'
+  - 'is_authorized'
+
+- subtitle: NetCDF Attribute functions
+  desc: Get a data.frame of attributes from a NetCDF object
+  contents:
+  - 'get_ncdf4_attributes'
+  - 'get_token'
+  - 'is_token_expired'
+  - 'get_mn_base_url'
+  - 'is_authorized'
+
+- subtitle: Sysmeta Function
+  desc: Function to access the sysmeta of the package
+  contents:
+  - 'get_all_sysmeta'
+
+
+- subtitle: Editing Functions
+  desc: Functions to edit a package
+  contents:
+  - 'publish_object'
+  - 'update_object'
+  - 'publish_update'
+  - 'create_resource_map'
+  - 'update_resource_map '
+  - 'set_file_name'
+  - 'reformat_file_name'
+
+- subtitle: Helper Functions
+  desc: Functions to help test a package
+  contents:
+  -  starts_with("create_")
+  - 'get_orcid_name'
+  - 'get_orcid_email'
+
+- subtitle: EML Functions
+  desc: Functions for creating and changing EML metadata
+  contents:
+  -  starts_with("pid_")
+  -  starts_with("eml_")
+  - 'sysmeta_to_eml_physical'
+  - 'get_orcid_email'
+  - 'set_abstract'
+  - 'which_in_eml'
+  - 'reorder_pids'
+
+- subtitle: Environment Functions
+  desc: Functions related to loading configuriation based upon the environment
+  contents:
+  -  starts_with("env_")
+
+- subtitle: Format Functions
+  desc: Functions related to data object formats
+  contents:
+  - 'guess_format_id'
+  - 'format_eml'
+  - 'format_iso'
+  - 'format_eml'
+
+  - subtitle: Quality Function
+  desc: Function to check the quality of an EML document you are authoring
+  contents:
+  - 'mdq_run'
+
+
+  - subtitle: ISO Metadata Functions
```

Comment:
I had some issues running `pkgdown::build_site()` to see a preview of what this will look like. THe issue should be coming from somewhere around here